crates/network/src/lib.rs

```diff
@@ -72,6 +72,9 @@ pub trait TransactionResponse {
 
     /// Gas limit
     fn gas(&self) -> u128;
+
+    /// Input data
+    fn input(&self) -> Bytes;
```

Comment:
perhaps we should change this to &[u8] to match:

https://github.com/alloy-rs/alloy/blob/5eadbf18c468d6b77aa0f3210b80245aa431f3f4/crates/consensus/src/transaction/mod.rs#L58-L58

but I could see how Bytes would be more useful in-case you want ownership.

wdyt @DaniPopes

Reply:
I realized that `input` type of `alloy_rpc_types_eth::Transaction` (`Bytes`) is different from `alloy_consensus::Transaction` (`&[u8]`). 
https://github.com/alloy-rs/alloy/blob/b79257c4de39b2486adc78ecb12cc72fdcb5253e/crates/rpc-types-
eth/src/transaction/mod.rs#L76

I set `Bytes` for the `input` type to match the `alloy_rpc_types_eth::Transaction` type in this PR since `TransactionResponse` is defined as  `alloy_rpc_types_eth::Transaction` for ethereum network:
https://github.com/alloy-rs/alloy/blob/main/crates/network/src/ethereum/mod.rs#L25

Should we fix that as well to match the type?